src/components/ScrollView.tsx

```diff
@@ -42,8 +44,11 @@ export const ScrollView: React.FC<Props> = ({
         contentInset={{ top: 0, bottom: 0 }}
         automaticallyAdjustContentInsets={false}
         contentInsetAdjustmentBehavior="never"
+        overScrollMode="never"
```

Comment:
Fix related to the overscroll bug (app closing) https://github.com/react-native-webview/react-native-webview/issues/2364